.github/tests/validate_new_tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+missing_files=()
+for file in $@
+do
+    if grep -Fxq $file .github/tests/tests_passing_pylint.txt
+    then
+        :
+    else
+        missing_files+=($file)
+    fi
+done
+
+if [ ${#missing_files[@]} -eq 0 ]; then
+    echo "All new test files are already added"
+    exit 0
+fi
+
+echo "Please add the following test files to ~/.github/tests/tests_passing_pylint.txt:"
+for file in ${missing_files[@]}; do echo $file; done
+exit 1
```

Comment:
Can we validate edited test files too? Maybe with the option of skipping the validation in case the test file is huge and there is no time in solving the pylint issues.

Otherwise I don't think anyone will fix the current test files